CHANGELOG.md

```diff
@@ -4,6 +4,86 @@
 
 ### Added
 
+- **decidim-initiative**: Skip initiative type selection if there is only one initiative type. [\#5835](https://github.com/decidim/decidim/pull/5835)
```

Comment:
From this release there's no need to add changelog entries, you can remove them

Reply:
Hi Ivan! Yes, I just resolved this conflict in #6152.

This PR (6218) depends on #6152.